### PR TITLE
Use proper links and images

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Fixes the recent push to `master` that contained "untested" README badges.

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
Result is https://github.com/kerl/kerl/pull/424/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5:
image

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+kerl [![GitHub Actions CI][ci-img]][ci] [![GitHub Actions Lint][lint-img]][lint]
 ====
 
-[kerl]: https://github.com/kerl/kerl
-[ci-img]: https://github.com/kerl/kerl/workflows/ci/badge.svg
-[lint-img]: https://github.com/kerl/kerl/workflows/lint/badge.svg
+[ci-img]: https://github.com/kerl/kerl/actions/workflows/ci.yml/badge.svg
+[ci]: (https://github.com/kerl/kerl/actions/workflows/ci.yml)
+[lint-img]: https://github.com/kerl/kerl/actions/workflows/lint.yml/badge.svg
+[lint]: (https://github.com/kerl/kerl/actions/workflows/lint.yml)
```
Comment (paulo-ferraz-oliveira) [inline README.md:7]:
```suggestion
[ci-img]: https://github.com/kerl/kerl/actions/workflows/ci.yml/badge.svg
[ci]: https://github.com/kerl/kerl/actions/workflows/ci.yml
[lint-img]: https://github.com/kerl/kerl/actions/workflows/lint.yml/badge.svg
[lint]: https://github.com/kerl/kerl/actions/workflows/lint.yml
```

Comment (paulo-ferraz-oliveira) [reply README.md:7]: Fix links

**Comment by paulo-ferraz-oliveira** [comment]
https://github.com/kerl/kerl/actions/runs/3548008249/jobs/5958755479#step:2:394 is timing out. Would you restart the workflow, please? Thank you.

**Review by jadeallenx — APPROVED** [review]

**Pull request merged**